gloo/docs/gloo_routing/virtual_services/security/apikey_auth/_index.md

```diff
@@ -1,30 +1,52 @@
 ---
-title: ApiKey Auth
+title: API Keys
 weight: 40
 description: How to setup ApiKey authentication. 
 ---
 
-There are a few different ways to use Gloo External Auth. In certain cases – such as protecting routes with
-secure, long-lived UUIDs – it may be desirable to secure a set of routes with **ApiKey Auth**. Keep in mind
-that your routes are only as secure as your apikeys; securing apikeys and proper apikey rotation is
-up to the user, thus the security of the routes is up to the user.
+{{% notice note %}}
+{{< readfile file="static/content/enterprise_only_feature_disclaimer" markdown="true">}}
+{{% /notice %}}
 
-In **ApiKey Auth**, the Gloo VirtualService containing the routes can be configured with a label selector to identify
-multiple valid apikey secrets, or direct references to apikey secrets. When the virtual service configuration changes,
-or when any apikey secret changes, Gloo immediately updates the external auth server with the new configuration. On the
-request path, Envoy asks the external auth service to check the request; any request to a path on that virtual
-service must have a valid apikey in the `api-key` header or will be denied.
+{{% notice note %}}
+The API keys authentication feature was introduced with **Gloo Enterprise**, release 0.18.5. If you are using an earlier version, this tutorial will not work.
+{{% /notice %}}
 
-## Setup
+{{% notice warning %}}
+{{< readfile file="/static/content/extauth_version_info_note" >}}
+{{% /notice %}}
 
-{{% notice note %}}
-ApiKey auth is a feature of **Gloo Enterprise**, release 0.18.5+. If you are using Open Source Gloo, this tutorial will not work.
+Sometimes when you need to protect a service, the set of users that will need to access it is known in advance and does 
+not change frequently. For example, these users might be other services or specific persons or teams in your organization. 
+You might also want to retain direct control over how credentials are generated and when they expire. If one of these 
+conditions applies to your use case, you should consider securing your service using 
+[API keys](https://en.wikipedia.org/wiki/Application_programming_interface_key). API keys are secure, long-lived UUIDs 
+that clients must provide when sending requests to a service that is protected using this method. 
+
+{{% notice warning %}}
+It is important to note that **your services are only as secure as your API keys**; securing API keys and proper API key 
+rotation is up to the user, thus the security of the routes is up to the user.
 {{% /notice %}}
 
+To secure your services using API keys, you first need to provide Gloo with your secret API keys in the form of `Secrets`. 
+After your API key secrets are in place, you can configure authentication on your Virtual Services by referencing the 
+secrets in one of two ways:
+
+1. you can specify a **label selector** that matches one or more labelled API key secrets (this is the preferred option), or
```

Comment:
TIL 💡 labelled (British) and labeled (American) are both valid spellings of the word